### DISCUSSION TITLE
Support for setting the correct timezone on the Create3

### DISCUSSION THREAD

**Discussion by jarkenau** [iRobotEducation/create3_docs#396] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

G.4.5

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

I used the [Setup NTP on compute board to serve time to Create® 3](https://iroboteducation.github.io/create3_docs/setup/compute-ntp/) tutorial to get the correct time in UTC on the Create3. Now, is there currently a way to use other timezones that UTC, e.g. CET (Central European Time), otherwise the logs are out of sync with the local time (in my case 2h behind).

**Comment by kwujciak** [comment]
Hello,
You can try going to your Create's webserver (its IP) to [set the date and time](https://iroboteducation.github.io/create3_docs/webserver/set-datetime/). Navigate to the "Beta Features" tab and click "Set Date and Time." Here, you can manually change the date and time.

**Reply by jarkenau** [reply]
This only works until you reboot the Create3. When it restarts, it will get the UTC time from an NTP server and overwrite your manual entry.

**Comment by richardw347** [comment]
Can I add support for this for another reason also, when debugging on a remote computer, I also had to set my computer time to CET otherwise the timestamps aren't in sync.
